src/lib/schematics/ng-add/setup-project.ts

```diff
@@ -14,7 +14,7 @@ import {
   getProjectStyleFile,
   hasNgModuleImport,
 } from '@angular/cdk/schematics';
-import {red, bold} from 'chalk';
+import {red, bold, italic} from 'chalk';
```

Comment:
I hadn't thought about this before, but is it a problem that `chalk` isn't a dependency of our packages? We went through this with `parse5`

Reply:
It's technically a dependency but `chalk` has been used before my changes AFAIK and we just assumed that `chalk` is always present due to the CLI coming with Chalk.

It might be better to be explicit though. Same applies for the `@schematics/angular` package we depend on.

Reply:
Actually, doesn't look like the CLI brings `chalk` in. It's required by:

```
    "/@babel/highlight",
    "/inquirer",
    "/postcss",
    "/speed-measure-webpack-plugin",
    "/tslint"
```

So kind of brittle. But the same applies for `tslint` (for `ng-update`) and the aforementioned `@schematics/angular` package. I can look in a follow-up for some API's the devkit provides for terminal colors (I think I saw something about that)